Firestore/Source/API/FIRFirestore.mm

```diff
@@ -217,6 +217,11 @@ - (void)setSettings:(FIRFirestoreSettings *)settings {
                             "longer be changed. You can only set settings before calling any "
                             "other methods on a Firestore instance.");
     }
+    if (!settings.enableTimestampsInSnapshots) {
```

Comment:
Any suggestions about the text are welcome. Also, I'm not sure if this warrants `FSTWarn`, or if it should be `FSTLog`.

Reply:
I think it needs to be FSTWarn to be visible-by-default, so I think this is right... And we may want to double-check that it really is visible-by-default when you run your app from xcode during development. The icore logging infrastructure is a bit nuanced I think...

Also, I think we probably want to move this check / log message to where we create FSTFirestoreClient (i.e. when the settings are used), since we want to log this message in the case they don't explicitly set the settings and just use the default ones.

Reply:
Done.